backend/routes/survey.py

```diff
@@ -15,7 +15,20 @@ async def create_survey(request: Request, survey: Survey = Body(...)):
     return JSONResponse(status_code=status.HTTP_201_CREATED, content=created_survey)
 
 
-@survey_router.get("/", response_description="Get all survey")
-async def get_survey(request: Request):
-    # implement this later
-    return JSONResponse(content={"hi":"Neha"})
+@survey_router.get("/", response_description="Get all surveys")
+async def list_surveys(request: Request):
+    surveys = []
+    for survey in request.app.mongodb["Survey"].find().to_list():  # would i need to add pagination?
```

Comment:
...why?

This would work perfectly fine, no?
```
return request.app.mongodb["Survey"].find().to_list()
```

Reply:
if the list of surveys is too long, i'm guessing the app could crash because the response size would be too big

Reply:
I mean we can add a parameter to just set the number of surveys to 5 or 10 per call to avoid that right?